.changeset/giant-paws-retire.md

```diff
@@ -0,0 +1,5 @@
+---
+'contexture-client': minor
+---
+
+Added `hasResults` prop for `results` and `pivot` nodes
```

Comment:
You also added it for every other node with a default function 👍

We should also add this to the documentation in the readme file